docs/source/rest.rst

```diff
@@ -5,16 +5,38 @@ Using bepasty's REST-API
 The Rest-API enables you to upload and download files, as well as retrieve informations
 about the file on the server.
 
-Currently (Version 0.3) the REST API provides four API Endpoints::
+Currently (Version |release|) the REST API provides the following API Endpoints::
```

Comment:
The docs are versioned with releases, so the version reference here could be also removed completely.